lxqt: init at 0.11

romildo opened the pull request:
###### Motivation for this change

Add [LXQt](http://lxqt.org/) (The Lightweight Qt Desktop Environment) to nixpkgs.
###### Things done
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

romildo committed:
lxqt: init at 0.11

romildo committed:
lxqt: standard patch applied to several lxqt packages

romildo committed:
libqtxdg: init at 2.0.0

romildo committed:
libsysstat: init at 0.3.2

romildo committed:
liblxqt: init at 0.11.0

romildo committed:
libfm-qt: init at 0.11.1

romildo committed:
lxqt-about: init at 0.11.0

romildo committed:
lxqt-admin: init at 0.11.0

romildo committed:
lxqt-common: init at 0.11.0

romildo committed:
lxqt-config: init at 0.11.0

romildo committed:
lxqt-globalkeys: init at 0.11.0

romildo committed:
lxqt-notificationd: init at 0.11.0

romildo committed:
lxqt-openssh-askpass: init at 0.11.0

romildo committed:
lxqt-l10n: init at 0.11.0

romildo committed:
lxqt-powermanagement: init at 0.11.0

romildo committed:
lxqt-qtplugin: init at 0.11.0

romildo committed:
lxqt-session: init at 0.11.0

romildo committed:
lxqt-sudo: init at 0.11.0

romildo committed:
pavucontrol-qt: init at 0.1.0

romildo committed:
qtermwidget: init at 0.7.0

romildo committed:
lxqt-panel: init at 0.11.0

romildo committed:
lxqt-runner: init at 0.11.0

romildo committed:
pcmanfm-qt: init at 0.11.1

romildo committed:
qterminal: init at 0.7.0

romildo committed:
qterminal: init at 0.7.0

romildo committed:
lxqt-policykit: init at 0.11.0

romildo committed:
lxqt service: init at 0.11

romildo committed:
lxqt: fix small issues in nix expressions

joachifm commented:
Wow, this is a pretty huge addition :) Looks pretty good to me.

romildo commented:
I have tested on a VM using a minimal configuration and it works for me. As I am not yet using it extensively some issues may appear. Once merged it will be easier to do more extensive tests.

The LXQt optional packages (besides `qterminal`) will be added after merging.

abbradar commented:
Nice! I had once an attempt to bring LXQt to NixOS but was somewhat disappointed in the end (I think multi-monitor support was somehow broken at that time) and abandoned it.

Two small issues from 5 minute testing in a VM:

* No start menu icon (is this perhaps intended?);
* `lxqt-config-monitor` doesn't work.

romildo committed:
lxqt: Add qtsvg to systemPackages

romildo committed:
lxqt: Fix mispelling of the word 'graphics'

romildo commented:
@abbradar I have fixed the missing panel icon for the main menu plugin.

romildo committed:
lxqt: Add libkscreen to systemPackages

romildo commented:
@abbradar I have also fixed the launching of the _Monitor Settings_ dialog (`lxqt-config-monitor`).

romildo committed:
pcmanfm-qt: use version from the LXQt desktop

pcmanfm-qt is part of the LXQt desktop. With the addition of LXQt to
nixpkgs, including a newer package for pcmanfm-qt, the older one is
removed and the entry in all-packages refers to the one in lxqt.

dezgeg commented on pkgs/top-level/all-packages.nix:
```diff
@@ -13702,8 +13702,6 @@ in
   pcmanfm = callPackage ../applications/misc/pcmanfm { };
 
-  pcmanfm-qt = callPackage ../applications/misc/pcmanfm-qt { };
-
```
Please add `pcmanfm-qt = lxde.pcmanfm-qt;` to keep old user configs working.

romildo replied:
@dezgeg Done.

romildo committed:
compton-conf: init at 0.2.0

romildo committed:
lxqt-common: Do not preserve mode when copying files to home direcotry

romildo committed:
obconf-qt: init at 0.11.0

bjornfor commented:
@romildo: Great stuff! I just tested it on a laptop.

It seems the screensaver isn't working/installed:

```
An error occurred starting screensaver. Action
'activate' failed. Ensure you have xscreensaver
installed and running.
```

Off topic, is there a way to let window borders be "thicker" than one pixel in openbox? I'm having a hard time resizing windows with my mouse.
